Engine/Component/ComponentText.cpp

```diff
@@ -74,7 +72,7 @@ void ComponentText::Render(float4x4* projection)
 		return;
 	}
 
-	glUseProgram(program);
+	program = App->program->UseProgram("UI Text");
```

Comment:
Getting the programs and have the use in the same method is cool, but in some case if you know that you are only going to use a program, going back to the map to look for that program every render cycle can be optimized by just saving which program are you going to use.

Reply:
That's right!